package.json

```diff
@@ -38,6 +56,18 @@
       "release": true
     },
     "plugins": {
+      "@release-it/bumper": {
+        "out": [
+          {
+            "file": "packages/react-native-nitro-image/package.json",
+            "path": "version"
+          },
+          {
+            "file": "example/package.json",
+            "path": "version"
+          }
+        ]
+      },
```

Comment:
This just also bumps the versions in `react-native-nitro-image` and `example`

Reply:
nice!